projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,37 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: inline-block;
+    border-color: var(--tui-base-02);
```

Comment:
I suppose it is unnecessary if you switch from border to outline.

Reply:
fixed